### PR TITLE
FIPS

### PR DESCRIPTION
**Pull request by cc7768**
This PR checks the current fips codes that are hardcoded and changes the sql where statements that are used to select a subset of counties to use the state code instead.

### CODE REVIEW

File: src/cmdc_tools/datasets/official/CA/counties/imperial.py [hunk 1]
```diff
@@ -28,7 +28,6 @@ class Imperial(DatasetBaseNoDate, ArcGIS):
     source = (
         "http://www.icphd.org/health-information-and-resources/healthy-facts/covid-19/"
     )
-    state_fips = 6
```
Comment (sglyon) [inline src/cmdc_tools/datasets/official/CA/counties/imperial.py]: Why is this removed here?

File: src/cmdc_tools/datasets/official/CA/counties/sandiego.py [hunk 2]
```diff
@@ -25,7 +25,6 @@ class SanDiego(DatasetBaseNoDate, ArcGIS):
         "https://www.sandiegocounty.gov/content/sdc/hhsa/programs"
         "/phs/community_epidemiology/dc/2019-nCoV/status.html"
     )
-    state_fips = 6
```
Comment (sglyon) [inline src/cmdc_tools/datasets/official/CA/counties/sandiego.py]: Why is this removed? I believe it will break the source logger

**Review by sglyon — CHANGES_REQUESTED** [review]

**Comment by cc7768** [comment]
I didn't think `state_fips` was being used by anything and thought it was an artifact of copy-paste -- I'll add it back since it sounds like it is used by logger.

File: src/cmdc_tools/datasets/official/CA/counties/imperial.py [hunk 3]
```diff
@@ -28,7 +28,6 @@ class Imperial(DatasetBaseNoDate, ArcGIS):
     source = (
         "http://www.icphd.org/health-information-and-resources/healthy-facts/covid-19/"
     )
```
Comment (cc7768) [inline src/cmdc_tools/datasets/official/CA/counties/imperial.py]:
```suggestion
    )
    state_fips = 6
```

File: src/cmdc_tools/datasets/official/CA/counties/sandiego.py [hunk 4]
```diff
@@ -25,7 +25,6 @@ class SanDiego(DatasetBaseNoDate, ArcGIS):
         "https://www.sandiegocounty.gov/content/sdc/hhsa/programs"
         "/phs/community_epidemiology/dc/2019-nCoV/status.html"
     )
```
Comment (cc7768) [inline src/cmdc_tools/datasets/official/CA/counties/sandiego.py]:
```suggestion
    )
    state_fips = 6
```

**Review by sglyon — APPROVED** [review]
